src/cli.h

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Please remove the metadata from the head, as it breaks consistency with the rest of the files
I understand that this might be auto-generated and/or was done in favor of Doxygen but for now, we omit this header